interview-questions.md

```diff
@@ -7,53 +7,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer:i belive is module value controler 
 
-  Researched answer:
+  Researched answer: Model view controller 
 
 
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: a data base written in Sql and yes there is 
 
-  Researched answer:
+  Researched answer: relational database stores and provides access to data points that are related to one another. yes there is non relational data bases. 
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer: ORM is a translator 
```

Comment:
Good opportunity here to show Who? What? When? Where? Why? and How? Even if we don't know the answer we can talk about our exp with similar concepts.